easybuild/easyconfigs/r/ReFrame/ReFrame-3.0.eb

```diff
@@ -0,0 +1,93 @@
+easyblock = 'PythonPackage'
+
+name = 'ReFrame'
+version = '3.0'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+
+sources = [{
+    'filename': 'v%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/eth-cscs',
+        'repo_name': 'reframe',
+        'tag': 'v%(version)s'
+    }
+}]
+
+checksums = ['8ecf8eeed11ad1d77d351c06d3c21775d62c16e4540bf66da8da0ed25b2004b2']
+
+# ReFrame requires Python 3.6, may only be available via 'python3'
+req_py_majver = 3
+req_py_minver = 6
+
+download_dep_fail = True
+use_pip = False
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
```

Comment:
This can be removed if you use `PythonBundle` (`download_dep_fail` is enabled by default in that case).